setup.py

```diff
@@ -7,7 +7,7 @@
 
 REQUIREMENTS = [
     'django-cms',
-    'html5lib>=0.999999999',
+    'html5lib>=1',
```

Comment:
Oh that's good to see. All those years of not being sure if you've counted enough 9s in the complex requirements between packages 😂